packages/agg/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/asciinema/agg"
+TERMUX_PKG_DESCRIPTION="asciinema gif generator"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.3.0"
+_COMMIT="43977bf11493650e76de0c9d4267bf0b4533255b"
+TERMUX_PKG_SRCURL="https://github.com/asciinema/agg/archive/${_COMMIT}.tar.gz"
```

Comment:
The Git repository for `agg` has version tags. Use

```
https://github.com/asciinema/agg/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```

as the source URL.

Note that the current value of `_COMMIT` is for head, not for `v1.3.0`.